app/models/lock_script.rb

```diff
@@ -25,9 +25,16 @@ def lock_info
         since_value = SinceParser.new(since).parse
         if since_value.present?
           tip_epoch = CkbUtils.parse_epoch(CkbSync::Api.instance.get_tip_header.epoch)
-          genesis_block = Block.find_by(number: 0)
-          genesis_block_time = DateTime.strptime(genesis_block.timestamp.to_s, "%Q")
-          estimated_unlock_time = genesis_block_time + (since_value.number * 4 + since_value.index * 4 / since_value.length).hours
+          block_interval = since_value.number * 1800 + since_value.index * 1800 / since_value.length - tip_epoch.number * 1800 + tip_epoch.index * 1800 / tip_epoch.length
```

Comment:
``` ruby
block_count = (since_value.number * 1800 + since_value.index * 1800 / since_value.length) - 
(tip_epoch.number * 1800 + tip_epoch.index * 1800 / tip_epoch.length)
```

Reply:
Please add a unit test for this, for others to understand this logic.

Reply:
fixed: https://github.com/nervosnetwork/ckb-explorer/pull/499/commits/5cc36e7f7631040f413432eb6a4b2808bba32bdf